src/features/import/components/Upload/UploadFile.tsx

```diff
@@ -0,0 +1,177 @@
+import { UploadFileOutlined } from '@mui/icons-material';
+import { useDropzone } from 'react-dropzone';
+import {
+  Box,
+  Button,
+  CircularProgress,
+  IconButton,
+  Typography,
+} from '@mui/material';
+import { useCallback, useState } from 'react';
+
+import messageIds from '../../l10n/messageIds';
+import theme from 'theme';
+import useImportedFile from '../../hooks/useImportedFile';
+import { Msg, useMessages } from 'core/i18n';
+
+const UploadFile = () => {
+  const [error, setError] = useState<boolean>(false);
+  const messages = useMessages(messageIds);
+  const { parseData, loading } = useImportedFile();
+
+  const onDrop = useCallback((acceptedFiles: File[]): void => {
+    acceptedFiles.map((file: File) => {
+      parseData(file);
+    });
+  }, []);
+
+  const { getRootProps, getInputProps, open, isDragActive } = useDropzone({
+    accept: {
+      types: ['.csv', '.xls', '.xlsx'],
+    },
+    maxFiles: 1,
+    noClick: true,
+    noKeyboard: true,
+    onDrop,
+    onDropAccepted: () => setError(false),
+    onDropRejected: () => setError(true),
+    onError: () => {
+      setError(true);
+    },
+  });
+
+  return (
+    <>
+      <Box
+        {...getRootProps()}
+        p={12}
+        style={
+          loading
+            ? {
+                alignItems: 'center',
+                backgroundColor: '#1976D214',
+                borderColor: '#1976D2',
+                borderRadius: 4,
+                borderStyle: 'dashed',
+                borderWidth: 2,
+                display: 'flex',
+                flexDirection: 'column',
+                flexWrap: 'wrap',
+                margin: 16,
+                textAlign: 'center',
+              }
+            : {
+                alignItems: 'center',
+                backgroundColor: error ? '#fdf7f7' : 'transparent',
+                borderColor: error ? theme.palette.error.dark : '#E0E0E0',
+                borderRadius: 4,
+                borderStyle: error ? 'solid' : 'dashed',
+                borderWidth: 2,
+                display: 'flex',
+                flexDirection: 'column',
+                flexWrap: 'wrap',
+                margin: 16,
+                textAlign: 'center',
+              }
+        }
+      >
+        {loading && (
+          <Box>
+            <CircularProgress sx={{ color: '#1976D2' }} />
```

Comment:
Color should come from theme. I think we can just use the default (i.e. no `sx` property at all) and if that doesn't match the design, that's a design system problem.

Reply:
Yes, color can't be default because then the loader will be red, and in the design all the loading stage is blue, that's why I added a color. I guess we need to align the design system to avoid those manual changes.

Reply:
Blue in figma is the default color, and in the live app that's red. I know that's confusing, but it's the same for all designs. The default blue in figma is intended to be default red in code.